test/smpte-timecode-test.js

```diff
@@ -102,6 +102,13 @@ describe('Constructor tests', function(){
     it ('non-standard frame rates', function() {
         expect(Timecode('00:10:00:00',28).frameCount).to.be(16800);
     });
+
+    it ('parses semicolon-delimited timecodes', () => {
+        const tc = Timecode('00;10;00;23', 29.97);
+        expect(tc.frameCount).to.be(18005);
+        expect(tc.dropFrame).to.be(true);
+        expect(tc.toString()).to.be('00:10:00;23');
```

Comment:
I do wonder if we should be converging on the `00:00:00;00` syntax if the user has been seeing it differently in their NLE. 

If it was me, I would expect Frame.io to show timecode the same way that my NLE did.

Reply:
@jhodges10 i tend to agree (caveat: i'm not an NLE user), but let's do that in a separate ticket! https://frame-io.atlassian.net/browse/CORE-4007

Reply:
Love it!